### ISSUE TITLE
egs/Demo_ZSTTS/meta directory missing

### ISSUE THREAD

**Issue by lenzo-ka** [constructor-tech/attest-speech-analysis-tool#1]
Looks like the directory `egs/Demo_ZSTTS/meta` is not checked in, or it's the wrong path:

```
FileNotFoundError: Filelist 'egs/Demo_ZSTTS/meta/filelist.txt' not found!

Traceback:

2024-09-10 10:05:27.254 Uncaught app exception
Traceback (most recent call last):
  File "/.../attest-speech-analysis-tool/ve_attest/lib/python3.10/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 552, in _run_script
    exec(code, module.__dict__)
  File "/.../attest-speech-analysis-tool/attest/ui/run.py", line 123, in <module>
    st.session_state.result = evaluate(
  File "/.../attest-speech-analysis-tool/attest/src/evaluate.py", line 30, in evaluate
    project = load_project(project)
  File "/.../attest-speech-analysis-tool/attest/src/model/internal_model.py", line 88, in load_project
    raise FileNotFoundError(error_msg)
FileNotFoundError: Filelist 'egs/Demo_ZSTTS/meta/filelist.txt' not found!

```

This error was raised after a clean install and setup through the web ui when clicking on "Home".

**Comment by dmitrii-obukhov** [comment]
Hi @lenzo-ka,

Thank you for reporting this, I will provide clearer error message.
The problem here is that Demo_ZSTTS is not a project, but a group of projects, so it does not contain meta directory. It should work if you specify group Demo_ZSTTS in "Settings", and than choose project(s) (StyleTTS, XTTSv2, etc.)

**Comment by lenzo-ka** [comment]
Thanks @dmitrii-obukhov -- it happens from a clean setup, so you may want to make the demo work without hitting this issue, _and_ making a clearer error message.

**Comment by dmitrii-obukhov** [comment]
I apologize for the delayed response. The logic for selecting groups and projects has been updated, and error handling has also been improved.

**Comment by dmitrii-obukhov** [comment]
Fixed in 80c259061de57d4dcbb4284092175dfe9f710726
